src/pages/public_registration/PublicSummaryContent.tsx

```diff
@@ -35,6 +36,22 @@ export const PublicSummaryContent = ({ registration }: PublicRegistrationContent
               {entityDescription.abstract}
             </Typography>
           )}
+          {entityDescription.alternativeAbstracts.und && (
+            <>
+              <Accordion elevation={0}>
+                <AccordionSummary expandIcon={<ExpandMoreIcon color="primary" />} sx={{ padding: '0' }}>
+                  <Typography variant="h3" color="primary">
+                    {t('registration.description.alternative_abstract')}
+                  </Typography>
+                </AccordionSummary>
+                <AccordionDetails sx={{ padding: 0, marginBottom: '2rem' }}>
+                  <Typography style={{ whiteSpace: 'pre-line' }} paragraph>
```

Comment:
Når vi først er her tror jeg kanskje jeg ville fjerne denne både her og fra over her... Ser ingen grunn til å ha den annet enn at det blir mer "luft" som egentlig ikke trengs her IMO..?
```suggestion
                  <Typography style={{ whiteSpace: 'pre-line' }}>
```

Reply:
fant ut det var best å beholde denne for å få litt space til `Beskrivelse` som kan være under?